lib/modules/manager/npm/post-update/index.spec.ts

```diff
@@ -219,6 +219,27 @@ describe('modules/manager/npm/post-update/index', () => {
       ]);
     });
 
+    it('writes .npmrc files', async () => {
+      await writeExistingFiles(updateConfig, additionalFiles);
+      expect(fs.writeLocalFile).toHaveBeenCalledOnce();
+      expect(fs.writeLocalFile).toHaveBeenCalledWith(
+        'packages/core/.npmrc',
+        '#dummy\n'
```

Comment:
why does this value comes from? can you make the test more self-contained please?

Reply:
It comes from the definition of `additionalFiles`, but I can definitely make smaller filesets contained within the test itself.